<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Liste des projets</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QPushButton" name="addProjectButton">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="removeProjectButton">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="projectView"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="importProjectButton">
          <property name="text">
           <string>Importer une liste</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Liste des demandes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="addDemandButton">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="removeDemandButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="demandView"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="ImportDemandButton">
          <property name="text">
           <string>Importer une liste</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="infosBox">
          <property name="title">
           <string>informations</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="titleLabel">
             <property name="text">
              <string>Titre de l'attribution :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="titreLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="informationLabel">
             <property name="text">
              <string>Informations additionnelles:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="infosTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="numberOfChoicesBox">
             <property name="title">
              <string>Nombre de choix</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QSpinBox" name="numberOfChoiceSpinBox">
                <property name="suffix">
                 <string> choix</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="attributionMethodBox">
             <property name="title">
              <string>Méthode d'attribution</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QComboBox" name="algosComboBox">
                <item>
                 <property name="text">
                  <string>directe</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>hongroise</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="VarGroupBox">
          <property name="title">
           <string>Variables additionelles</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Désolé, les variables additionnelles ne sont pas encore activées dans cette version du programme.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Résultats</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="exportPdfButton">
          <property name="text">
           <string>Exporter en pdf</string>
          </property>
          <property name="icon">
           <iconset resource="../rc/icons.qrc">
            <normaloff>:/ico/pdf_ico.svg</normaloff>:/ico/pdf_ico.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="exportHtmlButton">
          <property name="text">
           <string>Exporter en html</string>
          </property>
          <property name="icon">
           <iconset resource="../rc/icons.qrc">
            <normaloff>:/ico/html_ico.svg</normaloff>:/ico/html_ico.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="attributionView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;fichier</string>
    </property>
    <addaction name="actionOuvrir_un_projet"/>
    <addaction name="separator"/>
    <addaction name="actionImporter_une_liste_de_projets"/>
    <addaction name="actionImporter_une_liste_de_demandes"/>
    <addaction name="separator"/>
    <addaction name="actionEnregistrer_le_projet_courant"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="actionFermer_le_projet"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>&amp;edition</string>
    </property>
    <addaction name="actionRecr_e_la_table_d_attribution"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>aide</string>
    </property>
    <addaction name="action_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecr_e_la_table_d_attribution">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/ico/attr_table_ico.svg</normaloff>:/ico/attr_table_ico.svg</iconset>
   </property>
   <property name="text">
    <string>recréer la table d'attribution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOuvrir_un_projet">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/ico/open_ico.svg</normaloff>:/ico/open_ico.svg</iconset>
   </property>
   <property name="text">
    <string>ouvrir un projet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImporter_une_liste_de_projets">
   <property name="text">
    <string>importer une liste de projets</string>
   </property>
  </action>
  <action name="actionImporter_une_liste_de_demandes">
   <property name="text">
    <string>importer une liste de demandes</string>
   </property>
  </action>
  <action name="actionEnregistrer_le_projet_courant">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/ico/save_ico.svg</normaloff>:/ico/save_ico.svg</iconset>
   </property>
   <property name="text">
    <string>enregistrer le projet courant</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFermer_le_projet">
   <property name="text">
    <string>fermer le projet</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>quitter</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="icon">
    <iconset resource="../rc/icons.qrc">
     <normaloff>:/ico/save_under_ico.svg</normaloff>:/ico/save_under_ico.svg</iconset>
   </property>
   <property name="text">
    <string>enregistrer sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>à propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
